express-get-json/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+  const array = [];
+  for (const index in grades) {
+    array.push(grades[index]);
+  } res.json(array);
```

Comment:
if you want to use .json, it is better to use 
res.status(statusCode).json(array)

so that you can explicitly set the status code

Reply:
<img width="372" alt="image" src="https://user-images.githubusercontent.com/113648549/205764189-dc59af1c-d455-41ca-97c3-878389b21031.png">